modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -18,18 +19,140 @@ As an Operator author, your Operator must meet additional requirements to run pr
 // TODO: Include more info w/ better steps on how to do this:
 //* You must understand the {product-title} proxy configuration.
 
-For the CSV requirements, you can make the following changes as the Operator author.
+For the CSV requirements, you can make the following changes as the Operator author. The following procedure uses the Memcached Operator created in the Operator SDK tutorials for Go, Ansible, and Helm.
 
 .Prerequisites
 
 * An Operator project with a CSV.
 
 .Procedure
 
-. Use SHA references to related images in two places in the CSV for your Operator:
+. Set an environment variable, such as `RELATED_IMAGE_MEMCACHED`, for any additional images used by the Operator in the `config/manager/manager.yaml` file:
++
+.Example `config/manager/manager.yaml`
+[%collapsible]
+====
+[source,yaml]
+----
+...
+spec:
+  ...
+    spec:
+      ...
+      containers:
+      - command:
+        - /manager
+        ...
+        env:
+        - name: RELATED_IMAGE_MEMCACHED
+          value: "docker.io/memcached:1.4.36-alpine"
+----
+====
+
+. Pull the related images with digests by replacing hard-coded image references with environment variables in the following files:
+
+* For Go-based Operator projects, update the `controllers/memcached_controller.go` file with the following changes:
++
+.`controllers/memcached_controller.go` diff
+[%collapsible]
+====
+[source,diff]
+----
+...
+// deploymentForMemcached returns a memcached Deployment object
+...
+	Spec: corev1.PodSpec{
+        	Containers: []corev1.Container{{
+-			Image:   "memcached:1.4.36-alpine",
++			Image:   os.Getenv("RELATED_IMAGE_MEMCACHED"),
+			Name:    "memcached",
+			Command: []string{"memcached", "-m=64", "-o", "modern", "-v"},
+			Ports: []corev1.ContainerPort{{
+...
+----
++
+The `os.Getenv` function returns an empty string in the variable is not set. Set the environment variable before changing the file.
```

Comment:
Where do you set this environment variable? I don't see it. Please be clearer what it is in the code snippet by enclosing it in angled brackets and add a callout that's explicit about what the user's supposed to do.